shiny/session/_session.py

```diff
@@ -242,72 +243,86 @@ def verify_state(expected_state: ConnectionState) -> None:
             if conn_state != expected_state:
                 raise ProtocolError("Invalid method for the current session state")
 
-        try:
-            await self._send_message(
-                {"config": {"workerId": "", "sessionId": str(self.id), "user": None}}
-            )
+        with contextlib.ExitStack() as stack:
+            try:
+                await self._send_message(
+                    {
+                        "config": {
+                            "workerId": "",
+                            "sessionId": str(self.id),
```

Comment:
I know this is just due to indentation changes, but as long as you're in here, the `str()` is no longer needed.

```suggestion
                            "sessionId": self.id,
```